package.json

```diff
@@ -24,6 +25,8 @@
   "homepage": "https://github.com/mozilla/fxa-shared#readme",
   "devDependencies": {
     "chai": "4.1.2",
+    "eslint": "^4.19.1",
+    "eslint-plugin-fxa": "^1.0.0",
```

Comment:
@deeptibaghel could we please use the git hash here. For example : https://github.com/mozilla/fxa-content-server/blob/master/package.json#L129 (not sure if 41504c9dd30e8b52900c15b524946aa0428aef95 is latest)